libraries/core-react/src/components/Table/Cell/HeaderCell.tsx

```diff
@@ -1,44 +1,82 @@
 import * as React from 'react'
 import { ThHTMLAttributes } from 'react'
 import styled, { css } from 'styled-components'
-import { typographyTemplate, spacingsTemplate } from '@utils'
+import { typographyTemplate, spacingsTemplate, bordersTemplate } from '@utils'
 import { token as tablehead, TableHeadToken } from './HeaderCell.tokens'
 import { useTable } from '../Table.context'
 import { applyDensity } from './utils'
 
 type BaseProps = {
   token: TableHeadToken
-}
+} & Pick<React.AriaAttributes, 'aria-sort'>
 
-const Base = ({ token }: BaseProps) => {
+const Base = (props: BaseProps) => {
+  const { token } = props
   const { background, height, typography, spacings } = token
+  const activeToken = token.states.active
+  const ariaSort = props['aria-sort']
+  let sortStylingHover = css({})
+  let sortStylingActive = css({})
+
+  if (ariaSort) {
+    sortStylingHover = css`
+      &:hover {
```

Comment:
Ref. lack of hover state on touch devices, shouldn't we wrap in a media query here as well?

Reply:
Added :)